src/map/main/MapLiveRoutes.js

```diff
@@ -16,6 +16,9 @@ const MapLiveRoutes = () => {
 
   const history = useSelector((state) => state.session.history);
 
+  const reportWidth = useAttributePreference('mapLineWidth', 2);
+  const reportOpacity = useAttributePreference('mapLineOpacity', 1);
```

Comment:
You should rename these variable as well to match. This one is definitely incorrect because it's not a report, but for consistency just call all variables `mapLine...`.